xwiki-platform-core/xwiki-platform-office/xwiki-platform-office-test/pom.xml

```diff
@@ -33,7 +33,7 @@
   <description>XWiki Platform - Office - Test -Parent POM</description>
   <modules>
     <module>xwiki-platform-office-test-pageobjects</module>
-    <module>xwiki-platform-office-test-tests</module>
+    <module>xwiki-platform-office-test-docker</module>
```

Comment:
2 things:
* You should remove the `office-tests` profile from the pom (in -office/pom.xml) and replace it with the `integration-tests` one (and remove from https://dev.xwiki.org/xwiki/bin/view/Community/Building/#HUsingProfiles after it's been committed).
* You should add the `docker` profile and move -test-docker to that profile.